j0flintking02/autoMart_api
# feat(add PostgreSQL component): implement data persistence

#### What does this PR do?
- setup the Postgres database
- ensure data doesn't disappear on system restart
- implement tests for all components
#### How should this be manually tested?
- clone the repo and switch to the branch `ch-setup_postgres-166716468` and install dependencies using `npm install`
- add a .env file from the .env.sample or export `DATABASE_URL` and the `SECRET_KEY`
- create 2 databases automart2019 and automart2019_test
- start the server using `npm run prestart` and then `npm start`
- if the app starts successfully then the database connection was successful else connection error
- you can test any of the routes
- then check the database to confirm the input

#### What are the relevant pivotal tracker stories?
[Delivers #166716468]
